core/templates/dev/head/components/rule_editor_directive.html

```diff
@@ -28,8 +28,10 @@
     </div>
 
     <div ng-if="rule.type == 'FuzzyMatches'">
-      <classifier-panel rule-inputs="rule.inputs" on-training-data-deletion="onDeleteTrainingDataEntry(index)">
```

Comment:
It looks like you've renamed the classifier_panel_directive but now you're also not using it any more. Is that an intentional change? If we are not using the directive then we should delete it.